uber-build-config.sh

```diff
@@ -13,7 +13,7 @@ else
 fi
 
 # Points to the repo that contains the private keystore values. This repo is therefore also private.
-KEYSTORE_GIT_REPO="git@github.com:typesafehub/typesafe-keystore.git"
+KEYSTORE_GIT_REPO="https://github.com/typesafehub/typesafe-keystore.git"
```

Comment:
this shouldn't be changed, otherwise ssh agent is not working. At least not on my system. Did you manually type in your password?